csm/core/services/audit_log.py

```diff
@@ -55,12 +56,15 @@
             "{http_status} {error_code} {bytes_sent} {object_size} {total_time}"
             "{turn_around_time} {referrer} {user_agent} {version_id} {host_id}"
             "{signature_version} {cipher_suite} {authentication_type} {host_header}"
-        )
+        ),
+        "sortable_fields" : ['timestamp']
     }
 }
 
 COMPONENT_NOT_FOUND = "no_audit_log_for_component"
-
+CSM_AUDIT_LOG_MSG_NON_SORTABLE_COLUMN = "csm_audit_log_non_sortable_column"
```

Comment:
Remove CSM_AUDIT_LOG_MSG_NON_SORTABLE_COLUMN and S3_AUDIT_LOG_MSG_NON_SORTABLE_COLUMN as those are not being used.

Reply:
Hi @soniyamoholkar, those are being used in the validation of sortable columns and raising the InvalidRequest error.